PicPi/KeychainService.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  KeychainService.swift
+//  PicPi
+//
+//  Created by Mustafa T. Mohammed on 12/9/20.
+//
+import Foundation
+import Security
+/*
+ 
+ Example usage:
+
+ let int: Int = 555
+ let data = Data(  int.utf8)
+ let status = KeyChain.save(key: "MyNumber", data: data)
+ print("status: ", status)
+
+ if let receivedData = KeyChain.loadKey(  "MyNumber") {
+     let result  Int(decoding: receivedData, as: UTF8.self)
+
+     print("result: ", result)
+ }
+ 
+ */
+class KeyChain {
```

Comment:
Love the comment and example!

These are objc styled comments but Xcode can help with swift comments:
highlight the class name and press `cmd+opt+/`. this also works for methods and vars. Here's more on swift docs https://nshipster.com/swift-documentation/